-------------------------------------------------------------------------------
Test set: aston.repository.UserRepositoryImplTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 4.656 s <<< FAILURE! - in aston.repository.UserRepositoryImplTest
aston.repository.UserRepositoryImplTest.save  Time elapsed: 0.214 s  <<< ERROR!
aston.exception.RepositoryException: 
org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "users_pk"
  Подробности: Key (user_id)=(2) already exists.
	at aston.repository.impl.UserRepositoryImpl.save(UserRepositoryImpl.java:66)
	at aston.repository.impl.UserRepositoryImpl.save(UserRepositoryImpl.java:14)
	at aston.repository.UserRepositoryImplTest.save(UserRepositoryImplTest.java:69)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "users_pk"
  Подробности: Key (user_id)=(2) already exists.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2713)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2401)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:368)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:498)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:415)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:190)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:152)
	at aston.repository.impl.UserRepositoryImpl.save(UserRepositoryImpl.java:51)
	... 5 more

